src/box/xlog.c

```diff
@@ -1872,15 +1873,15 @@ xlog_cursor_find_tx_magic(struct xlog_cursor *i)
 	return 0;
 }
 
+/**
+ * Same as @xlog_cursor_next_tx, but no tx cursor is created. Tx is returned
+ * in @a buf.
+ */
 int
-xlog_cursor_next_tx(struct xlog_cursor *i)
+xlog_cursor_next_tx_raw(struct xlog_cursor *i, struct ibuf *buf)
```

Comment:
The function should be static. The name is confusing because the function without the "raw" suffix also returns "raw" data. Better use "impl".